src/ContainerRenameModal.jsx

```diff
@@ -46,7 +46,7 @@ const ContainerRenameModal = ({ container, version, updateContainerAfterEvent })
                     Dialogs.close();
                     // HACK: This is a workaround for missing API rename event in Podman versions less than 4.1.
                     if (version.localeCompare("4.1", undefined, { numeric: true, sensitivity: 'base' }) < 0) {
-                        updateContainerAfterEvent(container.Id, container.isSystem);
+                        updateContainer(container.Id, container.isSystem);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230705-100559-66516e12-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)